easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.048_mariadb_base_version.patch

```diff
@@ -0,0 +1,20 @@
+diff -ur DBD-mysql-4.048.orig/dbdimp.c DBD-mysql-4.048/dbdimp.c
```

Comment:
@smoors Please include a short description of the patch + mention author or upstream reference to source of the patch (and don't forget to update the checksum accordingly)